BitPlanner/Scenes/CraftPage.cs

```diff
@@ -143,6 +144,52 @@ private void CopyTreesAsText()
         DisplayServer.ClipboardSet(result.ToString().Trim());
     }
 
+    public void CopyTreesAsCSV()
+	{
+		var result = new StringBuilder();
+		var separator = new string('-', 64);
+
+		// Ny header enligt rekommendation
+		result.AppendLine("Item,Minimum Quantity,Maximum Quantity,In Stock");
+
+		foreach (var tab in _recipeTabs.GetChildren().Cast<RecipeTab>())
+		{
+			var root = tab.GetTreeRoot();
+			TraverseAndAppendCSV(root, result, 0); // Skicka med depth = 0
+			result.AppendLine(separator); // Separera träd
```

Comment:
```suggestion
			result.AppendLine("\n\n");
```

That will be a better separator for a spreadsheet I think.